src/libraries/System.ServiceProcess.ServiceController/tests/ServiceControllerTests.cs

```diff
@@ -78,6 +78,14 @@ public void Start_NullArg_ThrowsArgumentNullException()
             Assert.Throws<ArgumentNullException>(() => controller.Start(new string[] { null } ));
         }
 
+        [ConditionalFact(nameof(IsProcessElevated))]
+        public void Stop_FalseArg_WithDependentServices_ThrowsInvalidOperationException()
```

Comment:
Can we also add a test for a valid use of this new API? i.e. a test that doesn't throw an exception.

Reply:
I've added a test that calls Stop(true) and checks that the service and its dependents have been correctly stopped.